tests/dialects/test_builtin.py

```diff
@@ -246,7 +246,7 @@ def test_DenseIntOrFPElementsAttr_fp_type_conversion():
 def test_DenseIntOrFPElementsAttr_from_list():
     attr = DenseIntOrFPElementsAttr.tensor_from_list([5.5], f32, [])
 
-    assert attr.type == AnyTensorType(f32, [])
+    assert attr.type == AnyTensorType(f32, [1])
```

Comment:
My understanding is that this change shouldn't change the external functionality (except for speed). Should this have been changed?

Reply:
this change is reverted with the support for zero rank tensors